docs: mitigate logs duplication in fluentd (#2278)
SumoLogic/sumologic-kubernetes-collection: deploy/docs/Troubleshoot_Collection.md

[v2_3]: https://github.com/SumoLogic/sumologic-kubernetes-collection/releases/tag/v2.3.0
[storage_class]: https://kubernetes.io/docs/concepts/storage/storage-classes/
[security_context]: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

### Duplicated logs

> We observed than under some conditions logs duplication using fluentd can happen:

Reviewer: ```suggestion
We observed than under certain conditions, it's possible for FluentD to duplicate logs:
```

- there are several requests made of one chunk

> - one of those request is failing resulting in marking all batch as failed

Reviewer: ```suggestion
- one of those requests is failing, resulting in the whole batch being retried
```

> In order to mitigate that, please use [fluentd-output-sumologic] with `use_internal_retry` option.
> Please consider the following example:

Reviewer: ```suggestion
In order to mitigate this, please use [fluentd-output-sumologic] with `use_internal_retry` option.
See the following example:
```
